templates/landscape/install.html

```diff
@@ -622,7 +429,7 @@ <h3 class="p-heading--2">Configure</h3>
               <ol class="p-list--divided">
                 <li class="p-list__item">
                   <p class="u-no-padding--top">Define parameters in variables</p>
-                  <p>Self-hosted Landscape users should set LANDSCAPE_ACCOUNT_NAME as <code>standalone</code>, Landscape SaaS customers should specify their account name.</p>
+                  <p>Self-hosted Landscape users should set LANDSCAPE_ACCOUNT_NAME as <code>standalone</code>, Landscape SaaS customers should specify their <u>account name</u> set LANDSCAPE_FQDN as <code>landscape.canonical.com</code>.</p>
```

Comment:
`<u>account name</u>` should really be a hyperlink to: https://landscape.canonical.com/settings